<launch>

    <node name="setup_attitude_world" pkg="quad_controller" type="setup_attitude_world.sh" output="screen" />

    <node name="attitude_controller" pkg="quad_controller" type="attitude_controller_node.py" output="screen" >
      <rosparam command="load" file="$(find quad_controller)/config/attitude_controller_config.yaml" />
    </node>

    <node name="quaternion_to_euler_node" pkg="quad_controller" type="quaternion_to_euler_node.py" output="screen"/>

    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />

</launch>
